docs/static/breaking-changes-90.asciidoc

```diff
@@ -46,4 +46,21 @@ removed and their replacements.
 | truststore_type |<<plugins-outputs-http-ssl_truststore_type>>
 |=======================================================================
 
-====
\ No newline at end of file
+====
```

Comment:
Maybe put these before the HTTP output - I think it might make sense to order these alphabetically

(cc @karenzone)